Automattic/wp-calypso
# Editor: CPT: Manage non-hierarchical term edits using Redux state

Similar in approach to #5445.

- 83fe77e is from #5445 (dd970a4 in original PR)
- abac80b is from #5445 (e03050d in original PR)

### To test

Verify no functional changes when editing non-hierarchical CPT terms:

> Edit a post of a custom type that has one or more non-hierarchical taxonomies.  Ensure that existing taxonomies are loaded correctly in the `TokenField`:

> - values (terms assigned to the saved post)
> - suggestions (all terms from the site)

> Ensure that terms can be updated and saved as expected.

Test live: https://calypso.live/?branch=update/editor/cpt-term-token-field-redux
